corehq/apps/hqcase/utils.py

```diff
@@ -272,7 +272,7 @@ def update_case(domain, case_id, case_properties=None, close=False):
     context = {
         'case_id': case_id,
         'date_modified': json_format_datetime(datetime.datetime.utcnow()),
-        'user_id': '',
+        'user_id': 'system',
```

Comment:
@czue Not sure what the right approach is here.  This is the update_case util that automatic case closure uses.  Is it ok to leave the user_id blank, or should it be set to some value like "system"?  This user_id gets passed to the case block here: https://github.com/dimagi/commcare-hq/blob/set-user-id/corehq/apps/hqcase/templates/hqcase/xml/update_case.xml#L1

Also, this util will leave user_id blank when it calls `submit_case_blocks`: https://github.com/dimagi/commcare-hq/blob/set-user-id/corehq/apps/hqcase/utils.py#L28. Not sure if leaving that blank is ok or if we should set it to some value like "system" as well.